automation/taskcluster/upload_apk_nimbledroid.py

```diff
@@ -17,7 +17,7 @@
 
 def uploadApk(apk,key):
 	headers = {"Accept":"*/*"}
-	payload = {'auto_scenarios':'false'}
+	payload = {'auto_scenarios':'false', 'device_config': 'android7'}
```

Comment:
(no action need) This looks correct. To be explicit, because I'm not familiar with the libraries or the HTTP, I want to walk through my understanding:
- Ilya requested to add the curl argument `-F device_config="android7`
- Per the curl manpage, -F "lets curl emulate a filled-in form in which a user has pressed the submit button."
- Per the [`requests.post` guide](https://2.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests), the `payload` arg will turn the body text into `{"form": {"key": "value"}}`, which seems reasonable to me
- This file is called for uploads in release, beta, and nightly builds